py-utils/src/utils/msg_bus/analytical_example.py

```diff
@@ -0,0 +1,51 @@
+from message_bus.bus import Bus
```

Comment:
Move all the test / example code to test folder. py-utils/test/message_bus/

Reply:
Moved